### PR TITLE
Remove Old analysis_* database fields

### LINKED ISSUE #2945: Deprecate PropertyState's analysis_state field
## context
This is a vestigial field and confusing now that SEED has Analyses (which don't use this field)

## task
- remove the `analysis_state` field, the related enums, and anywhere it is used in SEED

### PR DESCRIPTION
**Pull request by nllong**
#### Any background context you want to provide?
For the BRICR project, `analysis_state`, `analysis_state_message`, `analysis_start_time`, and `analysis_end_time`, were used to have SEED act as a queue to run OpenStudio simulations. More recently, analysis functionality has been added to see where the requests are in a pipeline. Additional queue-type analysis needs to leverage this functionality now.

#### What's this PR do?
* Removes the `analysis_state`, `analysis_state_message`, `analysis_start_time`, and `analysis_end_time` from the PropertyState model
* Removes `analysis_state`, `analysis_state_message`, `analysis_start_time`, and `analysis_end_time` from Scenarios
* Remove endpoint to filter on analysis state
* Update unit tests

#### How should this be manually tested?
* Thorough monkey testing

#### What are the relevant tickets?
Resolves #2945

### CODE REVIEW

File: seed/migrations/0156_auto_20211104_1638.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            model_name='scenario',
+            name='analysis_state',
+        ),
+        migrations.RemoveField(
+            model_name='scenario',
+            name='analysis_state_message',
+        ),
+        migrations.AlterIndexTogether(
+            name='propertystate',
+            index_together={('import_file', 'data_state', 'merge_state'), ('import_file', 'data_state', 'source_type'), ('hash_object',), ('import_file', 'data_state')},
```
Comment (nllong) [inline seed/migrations/0156_auto_20211104_1638.py:63]: removes the (org, analysis_state) index.

File: seed/api/v2_1/views.py [hunk 2]
```diff
@@ -40,14 +39,13 @@ class PropertyViewFilterSet(FilterSet, OrgMixin):
     Advanced filtering for PropertyView sets version 2.1.
     """
     address_line_1 = CharFilter(field_name="state__address_line_1", lookup_expr='contains')
-    analysis_state = CharFilter(method='analysis_state_filter')
```
Comment (macintoshpie) [inline seed/api/v2_1/views.py:43]: This isn't going to break OEP right?

**Review by macintoshpie — APPROVED** [review]
Sweet! Tested locally and things are working as expected 👍  I had one question, see below

File: seed/api/v2_1/views.py (hunk 2 above)
Comment (nllong) [reply seed/api/v2_1/views.py:43]: no, it was strictly BRICR...

**Pull request merged**